server/src/test/java/org/opensearch/action/bulk/TransportBulkActionIndicesThatCannotBeCreatedTests.java

```diff
@@ -153,7 +154,12 @@ private void indicesThatCannotBeCreatedTestCase(
             null,
             new IndexingPressureService(
                 Settings.EMPTY,
-                new ClusterService(Settings.EMPTY, new ClusterSettings(Settings.EMPTY, ClusterSettings.BUILT_IN_CLUSTER_SETTINGS), null)
+                new ClusterService(
+                    Settings.EMPTY,
+                    new ClusterSettings(Settings.EMPTY, ClusterSettings.BUILT_IN_CLUSTER_SETTINGS),
+                    null,
+                    mock(MetricsRegistry.class)
```

Comment:
Instead of mock (if you are not using it further), you can simply use NoopMetricsRegistry.INSTANCE

Reply:
Updated in the revision.